src/Symfony/Component/Validator/Attribute/HasNamedArguments.php

```diff
@@ -11,6 +11,9 @@
 
 namespace Symfony\Component\Validator\Attribute;
 
+/**
+ * Hint the loader that some constraint options are required.
```

Comment:
We can add a link to https://symfony.com/doc/current/validation/custom_constraint.html

```suggestion
 * Hint the loader that some constraint options are required.
 *
 * @see https://symfony.com/doc/current/validation/custom_constraint.html
```

Reply:
Link added!